Pod/Assets/plugins/aiq-plugin-imaging/www/imaging.js

```diff
@@ -6,7 +6,7 @@ cordova.define('aiq-plugin-imaging', function(require, exports, module) {
         // Empty constructor
     }
     
-    Imaging.prototype.capture = function(idOrSettings, settingsOrNil) {
+    Imaging.prototype.capture = function (idOrSettings, settingsOrNil) {
         console.warn('aiq.imaging.capture() is deprecated, use navigator.camera.getPicture() isntead');
```

Comment:
It's not really part of this PR. But there's a typo in the last word here.. "isntead". =)